src/cli.rs

```diff
@@ -50,6 +53,32 @@ pub enum SubCommand {
     },
 }
 
+#[derive(Clap, Debug)]
+pub enum PrintStyle {
+    /// Output with `export` prefix, can be evaled in shell.
+    SetEnv,
+    /// Output as object with `key` and `value` attributes.
+    Json,
+    /// Output as object with `key` and `value` attributes.
+    Yaml,
+}
+
+impl core::str::FromStr for PrintStyle {
+    type Err = clap::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match &s.to_lowercase()[..] {
```

Comment:
Personally, I'd skip the `to_lowercase` and allow this to fail with mixed case input

Reply:
Alright, let them burn.